src/core/Akka.Remote/Serialization/MessageContainerSerializer.cs

```diff
@@ -82,16 +75,30 @@ private Selection.Builder BuildPattern(string matcher, PatternType tpe)
             return builder;
         }
 
-        private byte[] SerializeActorSelectionMessage(ActorSelectionMessage sel)
+        private byte[] SerializeSelection(ActorSelectionMessage sel)
         {
             SelectionEnvelope.Builder builder = SelectionEnvelope.CreateBuilder();
-            Serializer serializer = system.Serialization.FindSerializerFor(sel.Message);
-            builder.SetEnclosedMessage(ByteString.CopyFrom(serializer.ToBinary(sel.Message)));
-            builder.SetSerializerId(serializer.Identifier);
-            if (serializer.IncludeManifest)
+            var message = sel.Message;
+            Serializer serializer = system.Serialization.FindSerializerFor(message);
```

Comment:
Again, could you help me understand these changes? Any idea if these have an impact on the binary compatibility of wire formats?